src/service/service.ts

```diff
@@ -643,6 +682,16 @@ export class Discv5 extends (EventEmitter as { new (): Discv5EventEmitter }) {
     }
   };
 
+  private onTalkReq = (srcId: NodeId, src: Multiaddr, message: ITalkReqMessage): void => {
+    log("Received TALKREQ message from Node: %s", srcId);
+    this.emit("talkReqReceived", srcId, message);
```

Comment:
I wonder if all available data should be emitted.
eg: `this.emit("talkReqReceived", srcId, src, message);`
Then again, I'm not sure that the `Multiaddr` is very useful / actionable.
It seems more likely that the ENR will be more useful. :shrug: 
I guess this can be revisited later.